es6-shim.js

```diff
@@ -3585,7 +3582,7 @@
     });
   }
 
-  if (supportsDescriptors) {
+  if (!ReflectShims.defineProperty) {    // possible alternative: if (supportsDescriptors)
```

Comment:
`supportsDescriptors` is only true when get/set are available - but `ReflectShims.defineProperty` might be available in the same way `Object.defineProperty` might be - working fine for enumerable/configurable/writable data properties, but throwing for everything else. What does this fix?

Reply:
This change is a key point to solve #466 and other `overrideNative` related crash issues.

In WSH(polyfilled by core-js), `supportsDescriptors`(es6-shim) is false bacause of `if(!supportsAccessors && (hasGetter || hasSetter))`(es5-shim) is true.

Because of this, `replacement` parameter is undefined when calling `overrideNative`. This causes the wrong override.

Reply:
This part should be run regardless of the values in `supportsDescriptors`.

Reply:
I'm still a bit unclear on your explanation.

If you're in WSH, polyfilled by core-js, why are you running es5-shim and es6-shim at all? You're supposed to use core-js OR es-shims, not both.